skipper/runner.py

```diff
@@ -92,6 +93,12 @@ def handle_volumes_bind_mount(docker_cmd, homedir, volumes, workspace):
     volumes.extend(['%(homedir)s/.netrc:%(homedir)s/.netrc:ro' % dict(homedir=homedir),
                     '%(homedir)s/.gitconfig:%(homedir)s/.gitconfig:ro' % dict(homedir=homedir)])
 
+    # required for docker login
+    if os.path.exists('%(homedir)s/.docker/config.json' % dict(homedir=homedir)):
```

Comment:
https://github.com/Stratoscale/skipper/blob/upstream/skipper/runner.py#L140
there is special treatment for this mount. Maybe let's always add it?

Reply:
:+1: